### PR TITLE
Upgrade to vuetify v3.5.x

### PR DESCRIPTION
**Pull request by MohamedElmdary**
### Description
Upgrade to vuetify v3.5.x

### Changes
- chore: upgrade vuetify package from 3.1.x to 3.4.x
- fix: Remove files from labs as it's now stable in components
- fix: Loading spinner inside button component
- fix: Breadcrumb navigator
- fix: deployments table data
- fix: manage gateway table
- fix: k8s deployment table
- fix: contracts list table
- fix: caprover manage worker dialog table
- fix: public ips table 
- fix: user farms table
- fix: user nodes table
- fix: farm finder dialog table
- fix: turn <template v-if> into <div v-show> to solve unexpected bug
- fix: selection in table 
- feat: Add update method as prop in -actions slots
- fix: use update method to update caprover worker after deploy/delete

### Related Issues
- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1219

### Documentation PR

For UI changes, Please provide the Documetation PR on [info_grid](https://github.com/threefoldtech/info_grid)

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by Mahmoud-Emad — APPROVED** [review]
Good job

**Comment by Mahmoud-Emad** [comment]
Check the mode finder page, please 

- [Screencast from 05-02-2024 02:54:59 PM.webm](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/57001890/70e7c2fe-0fd7-44cb-8e27-bdfb3754436f)

File: packages/playground/src/components/list_table.vue [hunk 1]
```diff
@@ -113,7 +113,15 @@ export default {
       },
     );
 
-    return { selectedItems, onUpdateSelection };
+    function toggleSelect(item: any) {
```
Comment (0oM4R) [inline packages/playground/src/components/list_table.vue:116]: do we still use it?

**Review by 0oM4R — COMMENTED** [review]
do we change the style of toggles ? 
the current one 
image

the new one :
image

**Review by 0oM4R — DISMISSED** [review]
data in backend column should'n be like this 
image
this may not related to your changes, but the case in 2.4.0 is :
image

so please merge latest branch to this one

**Comment by AlaaElattar** [comment]
- The table shouldn't be like that 

image

**Comment by AlaaElattar** [comment]
- I deployed vm and nothing appeared in the table 

image

**Comment by AlaaElattar** [comment]
- no data in all table 

image

**Comment by MohamedElmdary** [comment]
> do we change the style of toggles ? the current one image
> 
> the new one : image

I think we should stick with ui changes as it doesn't affect ux

**Comment by zaelgohary** [comment]
Playground build is failing.

**Comment by MohamedElmdary** [comment]
> Playground build is failing.

yes after merging from development it appears that there is files still using old vuetify api so I did some updates to fix it

**Review by amiraabouhadid — APPROVED** [review]

**Comment by amiraabouhadid** [comment]
spinners are larger than button 
image

**Review by amiraabouhadid — COMMENTED** [review]
spinner size fixed 
image

**Review by zaelgohary — COMMENTED** [review]
All tables are not shown. 

image

Deployment table is missing. I think this goes for all solutions

image

image



Node finder table size is not right.

image


Link color is not added

image

Staging:
image


Get TFT fails although it's working on staging.
image

**Comment by zaelgohary** [comment]
As for the tables, issue was fixed by building.

**Comment by MohamedElmdary** [comment]
> All issues were fixed by building the project except for the Get TFT button.

@zaelgohary 
After investing in `get tft` button I found that issue related to this PR https://github.com/threefoldtech/tfgrid-sdk-ts/pull/2563/files#diff-fce083955bd2accb8ca8a9c35f2f4c1915853c058393a1b14e1ae26dad9aa2c0R43-R45

**Comment by zaelgohary** [comment]
> > All issues were fixed by building the project except for the Get TFT button.
> 
> @zaelgohary After investing in `get tft` button I found that issue related to this PR https://github.com/threefoldtech/tfgrid-sdk-ts/pull/2563/files#diff-fce083955bd2accb8ca8a9c35f2f4c1915853c058393a1b14e1ae26dad9aa2c0R43-R45

You may open an issue for it so it won't be discarded.

**Review by zaelgohary — APPROVED** [review]
I opened an [issue](https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2684) for the get TFT button. Good job, Rabei!

**Pull request merged**